packages/models-library/src/models_library/api_schemas_api_server/functions.py

```diff
@@ -0,0 +1,44 @@
+from typing import Annotated, Any, Literal, TypeAlias
+
+from models_library import projects
+from pydantic import BaseModel, Field
+
+FunctionID: TypeAlias = projects.ProjectID
```

Comment:
should it be a UUID or a ProjectID which happens to be a UUID?

Reply:
I'd prefer to keep it ID for now. Since UUID implies a real UUID, which it might not be in the future.
I could consider FunctionUID, but i tried to keep it consistent with ProjectID.

Reply:
Ok, I thought you were talking about the left hand of the statement. I put UUID now.